manifests/init.pp

```diff
@@ -39,11 +47,17 @@
 #    st2::revison: 11
 #
 class st2(
-  $version            = '0.13.2',
-  $revision           = '5',
-  $autoupdate         = false,
-  $mistral_git_branch = 'st2-0.13.1',
+  $version                  = '0.13.2',
+  $revision                 = '5',
+  $autoupdate               = false,
+  $mistral_git_branch       = 'st2-0.13.1',
+  $conf_dir                 = $::st2::params::conf_dir,
+  $conf_file                = "$::st2::params::conf_dir/st2.conf",
```

Comment:
when doing interpolation, you should enclose in {}.

"${::st2::params::conf_dir}/st2.conf"